utils/constants.js

```diff
@@ -20,12 +20,12 @@ export const daysOfWeek = [
 export const modesOfTransport = [
   "Bicycle",
   "Bus",
+  "Carpool",
+  "Motorbike",
+  "Taxi Ride",
   "Car",
-  "Motorbike/scooter",
-  "Taxi/ride share",
+  "Walk", 
   "Train/tram",
-  "Walking/running",
-  "Other",
```

Comment:
I don't think we should remove 'Other' as an option.
How will respondents respond if their transport method isn't a listed option?

Reply:
Others has no meaning, on how you calculate the carbon emissions. This has been a question too by civic in figma.